clients/java/signalr/src/test/java/HubConnectionTest.java

```diff
@@ -369,14 +370,39 @@ public void SendWithFiveParamsTriggersOnHandler() throws Exception {
         assertEquals(12, value5.get(), 0);
     }
 
+    @Test
+    public void ReceiveHandshakeResponseAndMessage() throws Exception {
```

Comment:
nit: `receiveHandshakeResponseAndMessage` (Java methods are `camelCase` usually, and the others are ;))